.circleci/config.yml

```
@@ -337,16 +383,20 @@ jobs:
             - .git
       - run:
          name: test-daikon-part2.sh
-         command: export ORG_GRADLE_PROJECT_jdkTestVersion=25 && ./checker/bin-devel/test-daikon.sh
+         command: export ORG_GRADLE_PROJECT_jdkTestVersion=25 && ./checker/bin-devel/test-daikon-part2.sh
+         no_output_timeout: "30m"
 
 
 ## I'm not sure why the guava_jdk11 job is failing (it's due to Error Prone).
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)